docs/shared/carthage-installation-panel.mdx

```diff
@@ -11,43 +11,45 @@ import {
 
 <h4>Set up your `Cartfile`</h4>
 
-Add `github "apollographql/apollo-ios"` to your Cartfile. 
+Add `github "apollographql/apollo-ios"` to your Cartfile.
 
 </ExpansionPanelListItem>
 <ExpansionPanelListItem number="2">
 
-<h4>Checkout and build dependencies</h4>
+<h4>Check out and build dependencies</h4>
 
-Run `carthage update --platform ios` (or `--platform ios,macos` to build both Mac and iOS). **NOTE:** There's an issue with the way Carthage uses Lipo in the Xcode 12 GM, please cd into `[YourProject]/Carthage/Checkouts/apollo-ios/scripts` and then run `./carthage-build-workaround.sh` to actually get Carthage builds that work.
+Run `carthage update --platform ios` (or `--platform ios,macos` to build both Mac and iOS).
+
+> **Note:** There's an issue with the way Carthage uses Lipo in the Xcode 12 GM. Please `cd` into `[YourProject]/Carthage/Checkouts/apollo-ios/scripts` and then run `./carthage-build-workaround.sh` to resolve this build issue.
```

Comment:
Ah this is good that we're taking a look at this - Carthage has updated to build frameworks and I think we need to update the command to pass `--use-xcframeworks`